nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml

```diff
@@ -17,6 +17,9 @@ jobs:
           {% raw %}export NXF_VER=${{ matrix.nxf_ver }}{% endraw %}
           wget -qO- get.nextflow.io | bash
           sudo mv nextflow /usr/local/bin/
+      - name: Pull image
+        run: |
+          docker pull {{ cookiecutter.name_docker }}:dev && docker tag {{ cookiecutter.name_docker }}:dev {{ cookiecutter.name_docker }}:dev
       - name: Run test
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test,docker
```

Comment:
```suggestion
          nextflow run ${GITHUB_WORKSPACE} -ansi-log false -profile test,docker
```

Reply:
I think it's nicer to do this as a global environment variable (`NXF_ANSI_LOG=false`). Then it's for all test commands if there are multiple, and they're easier to read.

Reply:
Why do I remember seeing ANSI log even though that was set?... obviously completely missed that variable being set as an export which is why I added it in to the command. See your point though. Let me see if I can find an actual link showcasing my madness.

Reply:
Actually, `NXF_VER` was set and not `NXF_ANSI_LOG` which is why I was still seeing ANSI log. If that doesn't showcase my madness I don't know what will. @ggabernet would you mind exporting that variable instead and removing it from the command please? I will refrain from making any more comments on this PR, and rather take myself to bed :expressionless:

Reply:
hey sorry, I oversaw this, but will add it now